pkg/cmd/service/service_test.go

```diff
@@ -717,7 +722,7 @@ func TestHTTPServerEnabled(t *testing.T) {
 
 	ensureServiceUp(t, nil)
 
-	resp, err := http.Get("http://localhost:8080/healthz")
+	resp, err := retryablehttp.NewClient().Get("http://localhost:8080/healthz")
```

Comment:
Can we move this within `ensureServiceUp`?

Reply:
Done. Thanks!